### ISSUE TITLE
Possible Typo in Documentation: "Running minicap" Section Title

### ISSUE THREAD

**Issue by songyuc** [DeviceFarmer/minitouch#12]
Hello guys,

I was going through the minitouch documentation and noticed a potential typo in the section title. The section currently titled "Running minicap" seems to contain information relevant to "minitouch" rather than "minicap." 

Given the context and content, it appears that the section title might be a typographical error and could potentially be corrected to "Running minitouch." This would align the title with the information presented in the section.

This could be found in the README.md file or directly at [this link](https://github.com/DeviceFarmer/minitouch?tab=readme-ov-file#running-minicap).

Thank you for your attention to this matter, and for your continuous efforts in maintaining this valuable project.

Best regards,
Yucheng Song

**Comment by denis99999** [comment]
@songyuc , many thanks, fixed in #13

**Comment by songyuc** [comment]
Hi, @denis99999,

As I am a beginner of `minitouch`, I'm unsure about the relevance of minicap to `minitouch`. Could you please re-examine the minicap references in the ["For Android 10 and up"](https://github.com/DeviceFarmer/minitouch?tab=readme-ov-file#for-android-10-and-up) section of the README.md file, and help check for any possible typos?

Thank you in advance for your assistance.

**Comment by denis99999** [comment]
@songyuc , you are right, I forgot some other errors, it is fixed in #15, thanks also to @pcrepieux for his review
